tensorflow/core/kernels/data/iterator_ops.h

```diff
@@ -93,8 +93,27 @@ class IteratorResource : public ResourceBase {
       iterator.reset(static_cast<DatasetBaseIterator*>(it.release()));
     }
 
+    std::shared_ptr<FunctionLibraryDefinition> GetFlibDef() { return flib_def; }
```

Comment:
We usually use lower snakecase for accessors, like output_dtypes/output_shapes above, e.g. flib_def() instead of GetFlibDef()

Reply:
addressed in the commit

Reply:
@aaudiber please review.